<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Executing testA]]>
    </line>
    <line>
      <![CDATA[Test:testA is Fail]]>
    </line>
    <line>
      <![CDATA[TestName:testA Status:FAIL]]>
    </line>
    <line>
      <![CDATA[PHOTO:./photo/Mon Jul 06 16_52_53 IST 2020.png]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="16699" started-at="2020-07-06T16:52:38 IST" finished-at="2020-07-06T16:52:54 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16699" started-at="2020-07-06T16:52:38 IST" finished-at="2020-07-06T16:52:54 IST">
      <class name="script.DemoA">
        <test-method status="PASS" signature="openApp(java.lang.String, java.lang.String)[pri:0, instance:script.DemoA@16022d9d]" name="openApp" is-config="true" duration-ms="15224" started-at="2020-07-06T16:52:38 IST" finished-at="2020-07-06T16:52:53 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[localhost]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method status="FAIL" signature="testA()[pri:0, instance:script.DemoA@16022d9d]" name="testA" duration-ms="10" started-at="2020-07-06T16:52:53 IST" finished-at="2020-07-06T16:52:53 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at script.DemoA.testA(DemoA.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Executing testA]]>
            </line>
            <line>
              <![CDATA[Test:testA is Fail]]>
            </line>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method status="PASS" signature="closeApp(org.testng.ITestResult)[pri:0, instance:script.DemoA@16022d9d]" name="closeApp" is-config="true" duration-ms="1419" started-at="2020-07-06T16:52:53 IST" finished-at="2020-07-06T16:52:54 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testA status=FAILURE method=DemoA.testA()[pri:0, instance:script.DemoA@16022d9d] output=Executing testA]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[TestName:testA Status:FAIL]]>
            </line>
            <line>
              <![CDATA[PHOTO:./photo/Mon Jul 06 16_52_53 IST 2020.png]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- script.DemoA -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
